packages/e2e-tests/specs/performance/performance.test.js

```diff
@@ -47,22 +47,39 @@ function isKeyUpEvent( item ) {
 	return isKeyEvent( item ) && item.args.data.type === 'keyup';
 }
 
+function isFocusEvent( item ) {
+	return isKeyEvent( item ) && item.args.data.type === 'focus';
```

Comment:
What about click?

Reply:
The click also triggers focus, I chose this after monitoring the events with the Chrome profiler and it's pretty visible that all the behavior happens on focus.